avocado/utils/kernel.py

```diff
@@ -207,4 +208,4 @@ def check_version(version):
     :type version: string
     :param version: version to be compared with current kernel version
     """
-    assert LooseVersion(os.uname()[2]) > LooseVersion(version), "Old kernel"
+    assert packaging.version.parse(os.uname()[2]) > packaging.version.parse(version), "Old kernel"
```

Comment:
To avoid having > 80 cols here, one suggestion is to slip this:

```python3
os_version = packaging.version.parse(os.uname()[2])
version = packaging.version.parse(version)
assert os_version > version, "Old kernel"
```

Reply:
Sure. I will get to it rightaway.

Reply:
> Sure. I will get to it rightaway.

Thanks, now it would be good to squash both commits in 1.

Reply:
Will a squash happen before merge or am I supposed to combine both the commits into one? :confused:

Reply:
If you don't mind, please squash it.

Reply:
I believe both of the commits are adding towards the change. The first being the one importing the modules and the other being to one limiting the line count to 80 or under. So, I am not quite able to justify why there's a need of squashing the commits to one. 

(The way I see it every PR can be squashed into a single commit on merge)